extension/chrome/elements/composer/composer-footer.ts

```diff
@@ -41,9 +41,9 @@ export class ComposerFooter extends ComposerComponent {
       return '';
     }
     if (/^[*-_=+#~ ]+$/.test(footerFirstLine)) {
-      return `<br>${sanitizedHtmlFooter}`;  // first line of footer is already a footer separator, made of special characters
+      return sanitizedHtmlFooter;  // first line of footer is already a footer separator, made of special characters
     }
-    return `<br>--<br>${sanitizedHtmlFooter}`; // create a custom footer separator
+    return `--<br>${sanitizedHtmlFooter}`; // create a custom footer separator
```

Comment:
Looks ok if the `<br>` was moved elsewhere

Also have to see if I have intended to always use `<br / >` instead, I vaguely remember wanting to do that but forgot.

Reply:
More importantly, investigate resulting formatting